htdocs/wp-content/themes/owc-formulieren/assets/scss/style.scss

```diff
@@ -51,6 +51,7 @@
   Blocks
 --------------------------------------------------------------*/
 @import 'blocks/tile/shared';
+@import 'blocks/news/shared';
 
 // @import 'nlds/denhaag/tokens';
```

Comment:
Zet deze maar aan, en de `@import 'nlds/denhaag/tokens-used'` hieronder weer uit. Dan hoeven we niet bij elk component de juiste css variables toe te voegen.